### DISCUSSION TITLE
what is the code challenge all about

### DISCUSSION THREAD

**Discussion by ghost-2362003** [uaanchorage/GSoC#10] [General]
i did not understand the meaning of code challenge 
can somebody tell me what is this code challenge all about
and how and where to access this code challenge??

**Comment by pradeeban** [comment]
The application period for GSoC 2024 is over and 4 contributors have been accepted. GSoC 2025 application will be next year. We will publish details about code challenges for GSoC 2025 in January 2025.

**Comment by Moses-Mk** [comment]
Hello,

I'm Moses Makuei, a 3rd Year undergrad student pursuing computer science, majoring in artificial intelligence and data science.

I am passionate about machine learning, its transformative impacts and the prospects of contributing to a work that will impact many lives for the better. Contributing to your work presents to me the opportunity of achieving all the above.

I have vast knowledge in machine learning, computer vision and deep learning and have completed various projects.

I would like to contribute to your repositories, raising issues, solving the minor issues so as to familiarize myself with the structure of your projects and work in preparation for GSoC'25.

Any guidance rendered will be greatly appreciated. Thanks

Best Regards,
Moses Makuei
